rest/taskana-rest-spring-example-common/src/test/java/pro/taskana/example/ldap/LdapTest.java

```diff
@@ -48,4 +52,40 @@ void should_ReturnFullDnForUser_When_AccessIdOfUserIsGiven() throws Exception {
     String dn = ldapClient.searchDnForAccessId("user-2-2");
     assertThat(dn).isEqualTo("uid=user-2-2,cn=users,ou=test,o=taskana");
   }
+
+  @Test
+  void should_ReturnAllUsersInUserRoleWithCorrectAttributes() {
```

Comment:
The configuration contains all individual users in addition to the group. (taskana.roles.user= cn=ksc-users,cn=groups,OU=Test,O=TASKANA | teamlead-1 | teamlead-2 | user-1-1 | user-1-2 | user-2-1 | user-2-2 | user-b-1 | user-b-2)

Are we sure, that the group resolving works correctly? Why do we have this configuration?

Reply:
yes, I have taken out some individual users to make sure the group resolve works